extensions/analyticsdx-vscode-templates/src/util/definitions.ts

```diff
@@ -76,7 +77,7 @@ export class JsonAttributeRelFilePathDefinitionProvider extends JsonAttributeDef
       location.previousNode &&
       location.previousNode.type === 'string' &&
       location.previousNode.value &&
-      this.patterns.some(location.matches)
+      this.patterns.some(p => locationMatches(location, p))
```

Comment:
Should you pass in `false` for the `exact` parameter, so it matches the existing behavior?

Reply:
Nope, I want the new exact match behavior here (like in almost all the places), since it's the correct thing.